.github/actions/check-module-version/action.yml

```diff
@@ -0,0 +1,61 @@
+name: "Check Module Version"
```

Comment:
About this file and the publish-module-update one I guess this was copied from an other module and it's always done the same way

Why not move this in a shared repositoy instead, https://github.com/PrestaShop/.github maybe?
Or if we don't want to miw addons logic in an open source repo, which makes sense, a similar repository on the PrestaShopCorp organisation?

There is alread a `PrestaShopCorp/github-action-clean-before-deploy` so either a new similar repository for one action, or another for multiple actions